source/develop/release-management/features/network/managed_mtu_for_vm_networks.html.md

```diff
@@ -0,0 +1,332 @@
+transported either
+* as plain Ethernet frames,
+* encapsulated in a VLAN or
+* encapsulated in [GENEVE][3] tunnels.
+
+### Plain Ethernet frames
+If the Ethernet frames of the logical network are transported as a plain Ethernet
+frames of the physical network, the MTU of the logical network must not be
+grater than the MTU of the physical network.
+
+### VLAN
+The same rule applies, if the Ethernet frames of the logical network is
+transported encapsulated in a VLAN, because the added VLAN header belongs to
+the Ethernet header.
+
+### Tunnels
+If the Ethernet frames of the logical network are encapsulated in GENEVE
+tunnels, the calculation of the MTU of the logical network is not that simple
+anymore. While the VLAN is on data link layer, the GENEVE tunnels are using the
+transport layer. This means that the hosts might not be in the same broadcast
+domain. Although [GENEVE][6] and [OVN][7] supports IP fragmentation, this is
+not used in oVirt's default configuration to avoid tunnels with hard to detect
+performance issues.
+Hence, a Path MTU (PMTU) Discovery in the mesh of all possible hosts, which
+could serve VMs with access to the logical network, would be required to
+determine the maximum size of the encapsulating packet.
+There is a common maximal MTU for all OVN networks used in a cluster, because
+all OVN networks are transported over the same tunnels.
+This implies the requirement to manage MTU in a centralized way.
+
+Subtracting the encapsulation overhead from the maximum size of the
+encapsulating packet results in the MTU of the logical network.
+
+OpenStack puts the [burden to the networking plugin][4] to calculate
+the [largest possible MTU size that can safely be applied][5] to VMs,
+but in oVirt there will be no automatic calculation of the MTU.
+
+### MTU individually per network
+MTU has to be managed individually for each network, because:
+* The MTU of the logical network are transported either as plain Ethernet
+  frames or encapsulated in a VLAN is individual for each network.
+* The MTU for the external logical networks depends on the MTU of the underlying
+  network and for localnets on the MTU of the connected logical network, too.
+  There might be no common underlying network for the tunnels and this way it
+  the common MTU is not obvious.
+An individual MTU per network requires that oVirt engine supports the MTU
+property of external networks:
+* On Importing a network created outside engine on a provider.
+* During automatic synchronization of external networks.
+This requires enhancements in ovirt-engine, [OpenStack Java SDK][11] and
+ovirt-provider-ovn.
+
+### Propagating the MTU into the VMs
+
+#### Set the MTU by DHCP
+The MTU can be propagated via DHCP into the VMs. Since oVirt implements L3
+functionality only for external logical network, this approach works only for
+external logical networks and if a subnet is defined for the logical network
```

Comment:
logical networks and if a subnet is defined for the logical network and the VM configures the network interface via DHCP
=>
logical networks with a subnet defined and the VM configuring the network interface via DHCP.

Reply:
done